### ISSUE TITLE
[BUG] Hue dimmer not doing anything

### ISSUE THREAD

**Issue by francois-pasquier** [xaviml/controllerx#399]
# Bug report

## Description

I am getting no action from ControllerX when pressing buttons on my hue dimmer v2

## Additional information

* Devices involved:
  * Model: [929002398602](https://xaviml.github.io/controllerx/controllers/929002398602) as Controller
  * Light: [8718699673147](https://www.zigbee2mqtt.io/devices/8718699673147.html#philips-8718699673147) as Light
* Integration: z2m through mqtt directly
* AppDaemon version: v4.1.0
* ControllerX version: v4.16.0
* HACS version: 1.18.0
* Home Assistant Core version: v2021.12.7

## AppDaemon app configuration

```yaml
myApp:
  module: controllerx
  class: Philips929002398602LightController
  controller: zigbee2mqtt/Interrupteur Salle de bain/action
  integration: mqtt
  light: light.lumiere_salle_de_bain
```

```yaml
appdaemon:
  latitude: 0
  longitude: 0
  elevation: 30
  time_zone: Europe/Paris
  plugins:
    HASS:
      type: hass
      ha_url: http://192.168.1.10:8123
      token: REDACTED
    MQTT:
      type: mqtt
      namespace: mqtt
      client_host: mosquitto
http:
  url: http://appdaemon:5050
admin:
api:
hadashboard:

```

## Logs

```text
appdaemon            | fetch https://dl-cdn.alpinelinux.org/alpine/v3.14/main/x86_64/APKINDEX.tar.gz
appdaemon            | fetch https://dl-cdn.alpinelinux.org/alpine/v3.14/community/x86_64/APKINDEX.tar.gz
appdaemon            | OK: 951 MiB in 73 packages
appdaemon            | 2021-12-31 03:34:31.697730 INFO AppDaemon: AppDaemon Version 4.1.0 starting
appdaemon            | 2021-12-31 03:34:31.697836 INFO AppDaemon: Python version is 3.9.6
appdaemon            | 2021-12-31 03:34:31.697917 INFO AppDaemon: Configuration read from: /conf/appdaemon.yaml
appdaemon            | 2021-12-31 03:34:31.697986 INFO AppDaemon: Added log: AppDaemon
appdaemon            | 2021-12-31 03:34:31.698064 INFO AppDaemon: Added log: Error
appdaemon            | 2021-12-31 03:34:31.698151 INFO AppDaemon: Added log: Access
appdaemon            | 2021-12-31 03:34:31.698262 INFO AppDaemon: Added log: Diag
appdaemon            | 2021-12-31 03:34:31.705218 INFO AppDaemon: Loading Plugin HASS using class HassPlugin from module hassplugin
appdaemon            | 2021-12-31 03:34:31.712235 INFO HASS: HASS Plugin Initializing
appdaemon            | 2021-12-31 03:34:31.712357 INFO HASS: HASS Plugin initialization complete
appdaemon            | 2021-12-31 03:34:31.712479 INFO AppDaemon: Loading Plugin MQTT using class MqttPlugin from module mqttplugin
appdaemon            | 2021-12-31 03:34:31.715144 INFO MQTT: MQTT Plugin Initializing
appdaemon            | 2021-12-31 03:34:31.715238 INFO MQTT: Using 'mqtt-client/status' as Will Topic
appdaemon            | 2021-12-31 03:34:31.715309 INFO MQTT: Using 'mqtt-client/status' as Birth Topic
appdaemon            | 2021-12-31 03:34:31.715381 INFO MQTT: Using appdaemon_mqtt_client as Client ID
appdaemon            | 2021-12-31 03:34:31.715733 INFO AppDaemon: Initializing HTTP
appdaemon            | 2021-12-31 03:34:31.715905 INFO AppDaemon: Using 'ws' for event stream
appdaemon            | 2021-12-31 03:34:31.717128 INFO AppDaemon: Starting API
appdaemon            | 2021-12-31 03:34:31.718248 INFO AppDaemon: Starting Old Admin Interface
appdaemon            | 2021-12-31 03:34:31.718398 INFO AppDaemon: Starting Dashboards
appdaemon            | 2021-12-31 03:34:31.723296 INFO HASS: Connected to Home Assistant 2021.12.7
appdaemon            | 2021-12-31 03:34:31.725853 INFO AppDaemon: App 'myApp' added
appdaemon            | 2021-12-31 03:34:31.726204 INFO AppDaemon: Found 1 total apps
appdaemon            | 2021-12-31 03:34:31.726379 INFO AppDaemon: Starting Apps with 1 workers and 1 pins
appdaemon            | 2021-12-31 03:34:31.726839 INFO AppDaemon: Running on port 5050
appdaemon            | 2021-12-31 03:34:31.728427 INFO MQTT: Connected to Broker at URL mosquitto:1883
appdaemon            | 2021-12-31 03:34:31.731124 INFO AppDaemon: Got initial state from namespace mqtt
appdaemon            | 2021-12-31 03:34:31.731288 INFO MQTT: MQTT Plugin initialization complete
appdaemon            | 2021-12-31 03:34:31.746679 INFO HASS: Evaluating startup conditions
appdaemon            | 2021-12-31 03:34:31.748508 INFO HASS: Startup condition met: hass state=RUNNING
appdaemon            | 2021-12-31 03:34:31.748620 INFO HASS: All startup conditions met
appdaemon            | 2021-12-31 03:34:31.758338 INFO AppDaemon: Got initial state from namespace default
appdaemon            | 2021-12-31 03:34:33.730143 INFO AppDaemon: Scheduler running in realtime
appdaemon            | 2021-12-31 03:34:33.731461 INFO AppDaemon: Adding /conf/apps to module import path
appdaemon            | 2021-12-31 03:34:33.731648 INFO AppDaemon: App initialization complete
```

```text
Info MQTT publish: topic 'zigbee2mqtt/Interrupteur Salle de bain', payload '{"action":"on_press","battery":89,"linkquality":255,"update":{"state":"idle"},"update_available":false}'
Info MQTT publish: topic 'zigbee2mqtt/Interrupteur Salle de bain', payload '{"action":"","battery":89,"linkquality":255,"update":{"state":"idle"},"update_available":false}'
Info MQTT publish: topic 'zigbee2mqtt/Interrupteur Salle de bain/action', payload 'on_press'
Info MQTT publish: topic 'zigbee2mqtt/Interrupteur Salle de bain', payload '{"action":"on_press_release","battery":89,"linkquality":255,"update":{"state":"idle"},"update_available":false}'
Info MQTT publish: topic 'zigbee2mqtt/Interrupteur Salle de bain', payload '{"action":"","battery":89,"linkquality":255,"update":{"state":"idle"},"update_available":false}'
Info MQTT publish: topic 'zigbee2mqtt/Interrupteur Salle de bain/action', payload 'on_press_release'
```

## Additional Context

So basically, I am getting the logs from z2m, I had even one automation working but for some reason, ControllerX doesn't react to my dimmer keypresses.

It didn't either work when I was using `integration: z2m`.

I am open to any suggestion to troubleshoot this!

I may have missed something really obvious, I have no clue what can be the issue.

**Comment by xaviml** [comment]
Hi @francois-pasquier,

I do not see in the logs the ControllerX initialisation. How did you install ControllerX? Did you restart AppDaemon after installing ControllerX?

Regards,
Xavi M.

**Comment by francois-pasquier** [comment]
I am a big dummy, I thought ControllerX was only a hass integration and could communicate with AppDaemon.

I have added it to my AppDaemon container, thank you :)
